src/psyclone/domain/nemo/transformations/__init__.py

```diff
@@ -44,8 +44,7 @@
     import CreateNemoPSyTrans
 
 # The entities in the __all__ list are made available to import directly from
```

Comment:
Since we're here, this comment isn't true. It's purely for auto-api documentation generation.

Reply:
Removed comment (and I hope tor remove this whole folder in the next PR)